REMLA_PROJECT/src/visualization/evaluation.py

```diff
@@ -22,6 +22,15 @@ def evaluation():
     print(f"Accuracy: {accuracy_score(y_test, y_pred_binary)}")
     sns.heatmap(confusion_mat, annot=True)
 
+    metrics_dict = {
+    "accuracy": round(accuracy_score(y_test, y_pred_binary),5),
+    "roc_auc": round(roc_auc_score(y_test, y_pred_binary),5),
+    "f1": round(f1_score(y_test, y_pred_binary),5)
+    }
+
+    with open("REMLA_PROJECT\\reports\\metrics.json", "w") as json_file:
```

Comment:
change it to `with open("REMLA_PROJECT/reports/metrics.json", "w", encoding="utf-8") as json_file:`